### PR TITLE
EIP-7549: Add aggregate attestation interfaces

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

Adds the following interfaces:
- `SignedAggregateAttAndProof`
- `AggregateAttAndProof`

Because of circular dependencies, these two interfaces and the `Attestation` interface have to be defined under `proto` directory. Since it's preferable to keep all attestation-related interfaces in one place, I moved the last interface, `AttesterSlashing`, to the `proto` directory as well. Because of name clashes, interfaces' names use `Att` instead of `Attestation`.

### CODE REVIEW

File: proto/prysm/v1alpha1/validator-client/keymanager.proto [hunk 1]
```diff
@@ -1,7 +1,6 @@
 syntax = "proto3";
 package ethereum.validator.accounts.v2;
 
-import "google/protobuf/wrappers.proto";
```
Comment (rkapka) [inline proto/prysm/v1alpha1/validator-client/keymanager.proto:4]: this is unused and the warning annoyed me

File: beacon-chain/monitor/process_block.go [hunk 2]
```diff
@@ -124,14 +124,14 @@ func (s *Service) processSlashings(blk interfaces.ReadOnlyBeaconBlock) {
...
-					"sourceEpoch2":       slashing.GetSecondAttestation().GetData().Source.Epoch,
-					"targetEpoch2":       slashing.GetSecondAttestation().GetData().Target.Epoch,
+					"attestationSlot1":   slashing.FirstAttestation().GetData().Slot,
+					"beaconBlockRoot1":   fmt.Sprintf("%#x", bytesutil.Trunc(slashing.FirstAttestation().GetData().BeaconBlockRoot)),
+					"sourceEpoch1":       slashing.FirstAttestation().GetData().Source.Epoch,
+					"targetEpoch1":       slashing.FirstAttestation().GetData().Target.Epoch,
+					"attestationSlot2":   slashing.SecondAttestation().GetData().Slot,
+					"beaconBlockRoot2":   fmt.Sprintf("%#x", bytesutil.Trunc(slashing.SecondAttestation().GetData().BeaconBlockRoot)),
+					"sourceEpoch2":       slashing.SecondAttestation().GetData().Source.Epoch,
+					"targetEpoch2":       slashing.SecondAttestation().GetData().Target.Epoch,
```
Comment (saolyn) [inline beacon-chain/monitor/process_block.go:134]: This is definitely more a nit but may improve readability here.
```
firstAttestation := slashing.FirstAttestation().GetData()
secondAttestation := slashing.SecondAttestation().GetData()

log.WithFields(logrus.Fields{
	"attesterIndex":      idx,
	"blockInclusionSlot": blk.Slot(),
	"attestationSlot1":   firstAttestation.Slot,
	"beaconBlockRoot1":   fmt.Sprintf("%#x", bytesutil.Trunc(firstAttestation.BeaconBlockRoot)),
	"sourceEpoch1":       firstAttestation.Source.Epoch,
	"targetEpoch1":       firstAttestation.Target.Epoch,
	"attestationSlot2":   secondAttestation.Slot,
	"beaconBlockRoot2":   fmt.Sprintf("%#x", bytesutil.Trunc(secondAttestation.BeaconBlockRoot)),
	"sourceEpoch2":       secondAttestation.Source.Epoch,
	"targetEpoch2":       secondAttestation.Target.Epoch,
}).Info("Attester slashing was included")
```

Comment (prestonvanloon) [reply beacon-chain/monitor/process_block.go:134]: +1, except I prefer `data1` and `data2`

**Review by saolyn — APPROVED** [review]
LGTM!

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**